src/DssSpell.t.base.sol

```diff
@@ -1030,56 +1030,77 @@ contract DssSpellTestBase is Config, DssTest {
         uint256 tinBps,
         uint256 toutBps
     ) internal {
-        uint256 tin = tinBps * WAD / 10000;
```

Comment:
changes for inside this test helper (i'm commenting on the first line of the function, see file or expanded diff for full view) are introduced because this did not cover the edgecase when the line is set to 0 in the spell and the outstanding debt is non-0 (did not happen before). Now it should cover for 3 cases:
  1. line is 0 and Art > line 
  2. line is not 0 and Art > line
  3. Art < line

Where Art - debt , line - debt ceiling